tsconfig.json

```diff
@@ -2,9 +2,12 @@
   "compilerOptions": {
     "baseUrl": ".",
     "esModuleInterop": true,
+    "module": "Preserve",
+    "moduleResolution": "Bundler",
     "paths": {
       "@/shared/*": ["./packages/shared/src/*"],
-      "@/swap/*": ["./packages/swap/src/*"]
+      "@/swap/*": ["./packages/swap/src/*"],
+      "vite": ["node_modules/vite"]
```

Comment:
maybe we need to install `vite` as a dependency instead?

Reply:
updated with comment